### PR TITLE
Make biohazard announcements more clear on what they are spawning with new annoucments!

### PR DESCRIPTION
**Pull request by Code-Lyoko0**
## What Does This PR Do

Changes the biohazard announcements to state what actual biohazard is spawning instead of using codes, making it clearer to new players or players who don't know the codes what is actually happening. Does not remove the old sounds so they can be kept for future use if needed.

## Why It's Good For The Game

While I know the biohazard levels have been around for a while, I have always felt they where unclear especially when I was a newer player. This change makes it clear what is actually spawning and what they should prepare for.

## Images of changes

https://github.com/ParadiseSS13/Paradise/assets/13879982/d71b91b1-3ff2-4b5b-ae69-853af12fb6d9

https://github.com/ParadiseSS13/Paradise/assets/13879982/97f31085-d520-484e-8e52-cafcc7bdc003

https://github.com/ParadiseSS13/Paradise/assets/13879982/da60f3f6-127b-4242-bb62-d7e2afce939a

## Testing

Triggered events and saw new messages and heard new announcements.

## Changelog
:cl: Code-Lyoko0
tweak: Changed biohazard announcement to fit new voice announcements.
soundadd: Added three new biohazard warning messages, making it clearer what is spawning instead of using codes.
/:cl:

### CODE REVIEW

**Comment by Fordoxia** [comment]
Finally, I can immediately tell if it's xenos or terrors instead of waiting for a radio callout to confirm.

File: code/modules/events/spider_terror.dm [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 /datum/event/spider_terror/announce(false_alarm)
 	if(successSpawn || false_alarm)
-		GLOB.major_announcement.Announce("Confirmed outbreak of level 3-S biohazard aboard [station_name()]. All personnel must contain the outbreak.", "Biohazard Alert", 'sound/effects/siren-spooky.ogg', new_sound2 = 'sound/AI/outbreak3.ogg')
+		GLOB.major_announcement.Announce("Terror Spider infestation decected aboard [station_name()]. All personnel must contain the outbreak.", "Biohazard Alert", 'sound/effects/siren-spooky.ogg', new_sound2 = 'sound/AI/outbreak_terror.ogg')
```
Comment (Henri215) [inline code/modules/events/spider_terror.dm:14]:
```suggestion
		GLOB.major_announcement.Announce("Terror Spider infestation detected aboard [station_name()]. All personnel must contain the outbreak.", "Biohazard Alert", 'sound/effects/siren-spooky.ogg', new_sound2 = 'sound/AI/outbreak_terror.ogg')
```
Typo decected

Comment (Code-Lyoko0) [reply code/modules/events/spider_terror.dm:14]: I am a FOOL

**Comment by FunnyMan3595** [comment]
> Grab someone and see if they can hear a difference, could be just me

There's a bit of a difference, but I think it's just part of the variance the voice uses to not sound so flat over the course of a sentence. So a slightly different sentence gave slightly different pronunciation.

**Review by Henri215 — APPROVED** [review]

**Review by S34NW — APPROVED** [review]

**Pull request merged**